Shared/Samples/Navigate route with rerouting/NavigateRouteWithReroutingView.Model.swift

```diff
@@ -122,6 +122,7 @@ extension NavigateRouteWithReroutingView {
             
             // Set up the data source's locations using a local JSON file.
             let jsonData = try Data(contentsOf: .sanDiegoTourPath)
+            // swiftlint:disable:next optional_data_string_conversion
             let jsonString = String(decoding: jsonData, as: UTF8.self)
```

Comment:
Here, just use `String.init(contentsOf:encoding:)`:

```suggestion
            let jsonString = try String(contentsOf: .sanDiegoTourPath, encoding: .utf8)
```